packages/notifications/src/elements/global-alert/utility.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { createContext, useContext } from 'react';
+
+import { TYPE, IGlobalAlertProps } from '../../types';
+
+export type GlobalAlertContextProps = Pick<IGlobalAlertProps, 'type'>;
+
+export const GlobalAlertContext = createContext<GlobalAlertContextProps>({ type: TYPE[3] });
```

Comment:
This is brittle (subject to array re-ordering) and unreadable (have to go reference `TYPE[3]`). Please prefer explicit `type: 'info'` usage. TypeScript will do what is needed to keep things in check.